daml-lf/archive/src/main/protobuf/com/daml/daml_lf_1_11/daml_lf.proto

```diff
@@ -11,7 +11,7 @@ import "com/daml/daml_lf_1_11/daml_lf_1.proto";
 
 message ArchivePayload {
   // this is number 3 for historical reasons -- we had
-  // DAML-LF v0 and v1 before we had minor versions.
```

Comment:
all the proto files in 
-  `daml-lf/archive/src/main/protobuf/com/digitalasset/daml_lf_1_6`
-  `daml-lf/archive/src/main/protobuf/com/digitalasset/daml_lf_1_7`
-  `daml-lf/archive/src/main/protobuf/com/digitalasset/daml_lf_1_8`
-  `daml-lf/archive/src/main/protobuf/com/daml/daml_lf_1_11/`

are immutable snapshot of the archive proto definitions. They should  not be changed.

only the ones in `daml-lf/archive/src/main/protobuf/com/daml/daml_lf_1_dev/` should be changed.

Reply:
Thanks for the pointer. Reverted.

Reply:
For the test to pass, you are still missing to revert the changes in
 - `daml-lf/archive/src/main/protobuf/com/daml/daml_lf_1_11/`

Since the files were frozen on Wednesday after the last SDK release, I am fine to include your changes. 
If you want to go this way, we will have to changed `daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/ProtoTest.scala`.
In this case, let me know and I will push myself the changes on top of your PR.

Reply:
I did revert 1.11 in a second commit. I'd rather not make any non-trivial changes in this PR.

Reply:
:+1:

Reply:
> are immutable snapshots. They should not be changed.

FYI, we have a similar situation in
https://github.com/digital-asset/daml/tree/main/ledger/participant-integration-api/src/main/resources/db/migration/postgres, where the files must not change. To ensure this, we have manually computed sha checksums of the immutable files and committed them (see the `.sql` + `.sha` file tuples), and we have a bazel test that checks whether the actual checksums are still the same. You could copy the mechanism if you are concerned about someone changing the files.

Reply:
We already have that test https://github.com/digital-asset/daml/blob/main/daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/ProtoTest.scala